.circleci/config.yml

```diff
@@ -31,6 +34,11 @@ jobs:
           name: lint
           command: make lint
 
+      - save_cache:
+          key: pre-commit-{{ checksum ".pre-commit-config.yaml" }}
+          paths:
+            - "/root/.cache"
```

Comment:
Maybe,

```suggestion
            - "/root/.cache/pre-commit"
```

as there are other programs that use `.cache` for a cache directory (pip, playwright, firefox, ...).

Reply:
I want to cache those things too (currently pip and black).